Gemfile

```diff
@@ -1,3 +1,3 @@
 source 'https://rubygems.org'
 
-gem 'psych', '=2.2.4', :platform => "ruby"
+gem 'psych', '=2.2.4', :platform => :ruby_20 # Allows safe loading in older Ruby
```

Comment:
`Psych.safe_load` doesn't seem to be used by the runners, is it needed by Rack or Rails maybe?
Otherwise, I guess we could drop this line.

Reply:
It's mostly there because RVM gives a repeated warning message about it. Removing it wouldn't be a problem.

Reply:
Interesting, do you happen to have one of the warning messages? It doesn't seem RVM's job to check the psych version at first sight.
In any case, I think it's OK to leave it as changed, now psych is only installed on MRI 2.0.

Reply:
Here's an example:

~~~bash
ubuntu@ip-172-31-58-231:~/rsb$ rvm use ruby 2.0.0-p0
Using /home/ubuntu/.rvm/gems/ruby-2.0.0-p0
YAML safe loading is not available. Please upgrade psych to a version that supports safe loading (>= 2.0).
~~~